render/mapping.go

```diff
@@ -50,15 +51,31 @@ func theInternetNode(m RenderableNode) RenderableNode {
 	if len(m.Adjacency) > 0 {
 		node.ID = IncomingInternetID
 		node.LabelMajor = InboundMajor
-		node.LabelMinor = RequestsMinor
+		node.LabelMinor = InboundMinor
 	} else {
 		node.ID = OutgoingInternetID
 		node.LabelMajor = OutboundMajor
-		node.LabelMinor = RequestsMinor
+		node.LabelMinor = OutboundMinor
 	}
 	return node
 }
 
+// RemapEndpointIDs remaps endpoints to have the right id format.
```

Comment:
This comment is unhelpful. What format does it expect them to have? what is the `right` format?